### PR TITLE
add ananyo maiti info

### PR DESCRIPTION
**Pull request by kshitijsingla**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- add ananyo maiti info
https://github.com/ananyo2012

### CODE REVIEW

**Comment by abishekvashok** [comment]
Hey pls see the contributing.md file pls don't merge your own PRs

**Comment by robbyoconnor** [comment]
@kshitijsingla -- you are not to merge your own Pull Requests.

**Comment by robbyoconnor** [comment]
@kshitijsingla why did you not add twitter? I'm sure they have twitter. What about their blog? This is why you shouldn't merge it. Please add these things. I will not be approving your task until you do. If you merge it again, I will unassign the task from you -- Yes I am angry with you.

File: _data/mentors.yml [hunk 1]
```diff
@@ -99,6 +99,10 @@
   blog: http://ajinkya007.in/
   twitter: wavare_ajinkya
 
+  name: Ananyo Maiti
+  facebook: ananyo.maiti
+  github: ananyo2012
+
```
Comment (robbyoconnor) [inline _data/mentors.yml:105]: You didn't include the image.

**Comment by robbyoconnor** [comment]
@kshitijsingla -- I reverted your merge. Do it correctly this time now. That's why we have code reviews.

**Pull request merged**